addons/uom/models/uom_uom.py

```diff
@@ -4,6 +4,36 @@
 from odoo import api, fields, tools, models, _
 from odoo.exceptions import UserError, ValidationError
 
+PROTECTED_XML_IDS = [
+    "product_uom_unit",
+
+    "product_uom_ton",
+    "product_uom_kgm",
+    "product_uom_gram",
+
+    "product_uom_day",
+
+    "product_uom_km",
+    "product_uom_meter",
+    "product_uom_cm",
+
+    "product_uom_litre",
+    "product_uom_cubic_meter",
+
+    "product_uom_lb",
+    "product_uom_oz",
+
+    "product_uom_inch",
+    "product_uom_foot",
+    "product_uom_mile",
+
+    "product_uom_cubic_inch",
+    "product_uom_cubic_foot",
+    "product_uom_floz",
+    "product_uom_qt",
+    "product_uom_gal",
+]
```

Comment:
why aren't all UoM coming from a module protected, instead of hardcoding a list?

Reply:
Hours and Dozen (moved to demo) are not protected :D.